add bva.dyndns.info blog

v-bulynkin opened the pull request.

v-bulynkin committed:
add bva.dyndns.info blog

schulle4u commented:
Thanks for your entry. 👍 How stable is the connection to your home server? In my experience projects like this often have problems. Therefore, it would be best if you could deliver your website via a reliable 24/7 web host. If this is not possible, please explain why, so that I can decide whether your entry is still suitable for this list. 🙂

v-bulynkin commented:
It's quite stable, it's online since 2017 and works 24/7.
I changed hardware (look [here](https://bva.dyndns.info/about/) at the last picture for the current configuration).
Datenstrom Yellow was installed in October 2023 and replaced Wordpress.
Sometimes, because it is self-hosted config, I install updates and reboot it but most of the time it's available.

schulle4u commented:
Thank you, that sounds OK to me.
